Solutions/Microsoft Windows SQL Server Database Audit/Hunting Queries/SQL-UserRemovedFromSecurityAdmin.yaml

```diff
@@ -1,6 +1,9 @@
 id: f35b879c-c836-4502-94f2-c76b7f06f02d
 name: User removed from SQL Server SecurityAdmin Group
 description: |
+     This hunting query identifies user removed from the SecurityAdmin group of SQL Server
```

Comment:
"This hunting query identifies user removed from the SecurityAdmin group of SQL Server. It relies on the SQLEvent KQL Parser function."